addons/binding/org.openhab.binding.homematic/README.md

```diff
@@ -402,6 +410,28 @@ sendCommand(Var_1, RefreshType.REFRESH)
 
 **Note:** adding new and removing deleted variables from the GATEWAY-EXTRAS Thing is currently not supported. You have to delete the Thing, start a scan and add it again. 
 
+**Deprecated configuration for Thing ...: no address property available. Using old style configuration, please update your things and item file(s)**
+
+You are using a old configuration in your things and item file(s), example:  
```

Comment:
an

Reply:
If you added a headline above, you could link it in forum posts or the openHAB 2.2 release notes later.

Reply:
> If you added a headline above, you could link it in forum posts or the openHAB 2.2 release notes later.

I'm not sure what you mean, do you have an example?

Reply:
Sorry. What I meant is, the paragraph is currently in the middle of a lot of text. If you added a headline above it it would rendered to an html anchor you can then link e.g. in a forum posting.